crates/bins/src/bin/datadog-static-analyzer.rs

```diff
@@ -251,18 +251,28 @@ fn main() -> Result<()> {
         // if there is no config file, we must read the rules from a file.
         // Otherwise, we exit.
         if rules_file.is_none() {
-            eprintln!("no configuration and no rule files specified. Please have a static-analysis.datadog.yml file or specify rules with -r");
-            print_usage(&program, opts);
-            exit(1);
+            println!("WARNING: no configuration file detected, getting the default rules from the Datadog API");
+            println!("Check the following resources to configure your rules:");
+            println!(
+                " - Datadog documentation: https://docs.datadoghq.com/code_analysis/static_analysis"
+            );
+            println!(" - Static analyzer repository on GitHub: https://github.com/DataDog/datadog-static-analyzer");
+            let rulesets_from_api =
+                get_all_default_rulesets(use_staging).expect("cannot get default rules");
+            let rules_from_api: Vec<Rule> = rulesets_from_api
+                .iter()
+                .flat_map(|v| v.rules.clone())
+                .collect();
+            rules.extend(rules_from_api);
+        } else {
+            let rulesets_from_file = get_rulesets_from_file(rules_file.clone().unwrap().as_str());
+            let rules_from_file: Vec<Rule> = rulesets_from_file
+                .context("cannot read ruleset")?
+                .iter()
+                .flat_map(|v| v.rules.clone())
+                .collect();
+            rules.extend(rules_from_file);
```

Comment:
Similar to the previous comment, this could be: 
```rs
 rules.extend(
    rulesets_from_file
        .context("cannot read ruleset")?
        .into_iter()
        .flat_map(|v| v.rules),
);
```